### PR TITLE
Update storage.md

### PR DESCRIPTION
**Pull request by seguler**

### CODE REVIEW

**Comment by seguler** [comment]
@zezha-msft

**Comment by seguler** [comment]
#sign-off

File: docs-ref-services/storage.md [hunk 1]
```diff
@@ -39,8 +41,7 @@ pip install azure-mgmt-storage
 
 ## Example
 ```python
-storage_client = CloudStorageAccount(storage_account_name, storage_key)
-blob_service = storage_client.create_block_blob_service()
+blob_service = block_blob_service = BlockBlobService(account_name, account_key)
```
Comment (zezha-msft) [inline docs-ref-services/storage.md]: redundant variable name

**Comment by seguler** [comment]
@sign-off

**Pull request merged**